modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormPortlet.java

```diff
@@ -218,17 +222,24 @@ protected void setRenderRequestAttributes(
 			_ddmFormInstanceRecordVersionLocalService, _ddmFormInstanceService,
 			_ddmFormInstanceVersionLocalService, _ddmFormRenderer,
 			_ddmFormValuesFactory, _ddmFormValuesMerger,
-			_ddmFormWebConfigurationActivator.getDDMFormWebConfiguration(),
-			_ddmStorageAdapterRegistry, _groupLocalService, _jsonFactory,
-			_npmResolver, _objectFieldLocalService,
-			_objectFieldSettingLocalService, _objectRelationshipLocalService,
-			_portal, renderRequest, renderResponse, _roleLocalService,
-			_userLocalService, _workflowDefinitionLinkLocalService);
+			_getDDMFormWebConfiguration(), _ddmStorageAdapterRegistry,
```

Comment:
@icgsbr could you please delete the `DDMFormWebConfigurationActivator _ddmFormWebConfigurationActivator` variable as it is no longer being used?? Thanks!